doc/sphinx-guides/source/developers/deployment.rst

```diff
@@ -103,6 +103,44 @@ Caveat Recipiens
 
 Please note that while the script should work well on new-ish branches, older branches that have different dependencies such as an older version of Solr may not produce a working Dataverse installation. Your mileage may vary.
 
+
+Migrating Datafiles from Local Storage to S3
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+A number of pilot Dataverse installations start on local storage, then administrators are tasked with migrating datafiles into S3 or similar object stores. The files may be copied with a command-line utility such as `s3cmd<https://s3tools.org/s3cmd>`. You will want to retain the local file hierarchy, keeping the authority (i.e. 10.5072) at the bucket "root."
+
```

Comment:
Just to to nitpick a little, should probably be "e.g. 10.5072" or "for example, 10.5072" instead of "i.e. 10.5072"?